test/integration/api/admin/api_spec.js

```diff
@@ -1043,6 +1043,8 @@ describe("api", function () {
 
     it('should restore different versions of a deleted asset', async function () {
       this.timeout(TIMEOUT.LARGE);
+      this.retries(3);
```

Comment:
There's already a set of retries for in line 144 in this file,
It seems like the retries are too fast, Maybe we need a waiting timer here instead?

Reply:
Why do you think the retries are too fast?

Reply:
Because we already do 3 retries, and this test fails right?

So if 3 retries don't work, it probably means we try too fast

Reply:
alright added wait time

Reply:
Approved, but please change the PR name :)